Bolts/src/bolts/Task.java

```diff
@@ -222,7 +223,7 @@ public void run() {
     }
 
     final Task<Void>.TaskCompletionSource allFinished = Task.create();
-    final ArrayList<Exception> causes = new ArrayList<Exception>();
+    final ArrayList<Throwable> causes = new ArrayList<Throwable>();
```

Comment:
Can we change this to exception and avoid the cast in `allFinished.setError`? 
We never actually catch `Throwable`, only `Exception`